src/styles/CookieAnimation.css

```diff
@@ -0,0 +1,96 @@
+.press {
+  animation: press 0.15s ease-in-out;
+}
+.small-size {
+  width: 300px;
+  height: 300px;
+}
+
+@keyframes press {
+  0% {
+    transform: scale(1);
+  }
+  50% {
+    transform: scale(0.96);
+  }
+  100% {
+    transform: scale(1);
+  }
+}
+
+
+.cookie-container {
+  position: relative;
+  z-index: 10;
+}
+
+
+.mini-cookie {
+  position: absolute;
+  width: 55px !important ;
+  height: 55px !important;
+  opacity: 1;
+  animation: fadeOut 0.8s ease-out, floatUp 0.8s ease-out;
+  pointer-events: none;
+}
+
+@keyframes fadeOut {
+  0% { opacity: 1; }
+  100% { opacity: 0; }
+}
+
+@keyframes floatUp {
+  0% { transform: translateY(0); }
+  100% { transform: translateY(-30px); }
+}
+
+
+
+.column.is-3 {
```

Comment:
you are ruining styles in rest of application potentially. It doesnt care that you import in cookie clicker this gets run in whole application. If you want it to be only for cookie clicker move to style scoped section of cookieclicker.vue